Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1

```diff
@@ -0,0 +1,46 @@
+class FloatingIp : BaseResourceModel {
+    [String] $Name
+    [String[]] $PoolFqName
+    [String] $Address
+    [String[][]] $PortFqNames
+
+    [String] $ResourceName = 'floating-ip'
+    [String] $ParentType = 'floating-ip-pool'
+
+    FloatingIp([String] $Name, [String[]] $PoolFqName, [String] $Address) {
+        $this.Name = $Name
+        $this.PoolFqName = $PoolFqName
+        $this.Address = $Address
+    }
+
+    [String[]] GetFqName() {
+        return ($this.PoolFqName + @($this.Name))
+    }
+
+    [Hashtable] GetRequest() {
```

Comment:
What is the purpose of GetRequest command? Is it used for all of CRUD actions? I think that floating ip address is read-only property, so sending it for update will cause an error.

Reply:
I checked, it does not. You can update the floating_ip_address with in existing floating-ip with request like this.

Reply:
Thanks for checking. It's weird, in schema this field is marked as CR so it misled me. 
Does API server allow changing the address? What if somebody tries to change the IP and then update the floating ip?

Reply:
Yeah, according to docs it is not field that can be passed when using PUT method (updating object). But I just did it, and even in WebUI IP changed and everything just worked fine :)